xray/services/report.go

```diff
@@ -0,0 +1,190 @@
+package services
+
+import (
+	"encoding/json"
+	"errors"
+	"fmt"
+	"github.com/jfrog/jfrog-client-go/artifactory/services/utils"
+	"github.com/jfrog/jfrog-client-go/auth"
+	"github.com/jfrog/jfrog-client-go/http/jfroghttpclient"
+	clientutils "github.com/jfrog/jfrog-client-go/utils"
+	"github.com/jfrog/jfrog-client-go/utils/errorutils"
+	"net/http"
+)
+
+const (
+	reportsAPI         = "api/v1/reports"
+	vulnerabilitiesAPI = reportsAPI + "/vulnerabilities"
+)
+
+// ReportService defines the Http client and XRay details
```

Comment:
Let's change "XRay" to "Xray" (let's make sure to fix in other places in this file as well).

Reply:
I have updated the readme with sample usage.